### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by tagomaru**
This was implemented to support `parity` for sidechain.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+stages:
+  - build
+  - test
+  - e2e
+  - release
+  - deployment
+
+variables:
+  DOCKER_HOST: tcp://localhost:2375/
+  BASE_IMAGE_NAME: priv-testnet
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml]: this would be "parity"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
-# parity
-PolySwarm test network using parity
+# Development Network Setup
+
```
Comment (mrtizmoatwork) [inline README.md:6]: this area should mention that this project is for parity client

File: launch_parity_local.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DIR=$HOME/.ethereum/priv_testnet
```
Comment (mrtizmoatwork) [inline launch_parity_local.sh]: this won't be "priv_testnet" anymore. should be "parity" or "parity_testnet" or something.

**Review by mrtizmoatwork — CHANGES_REQUESTED** [review]
a few minor changes

File: launch_parity_local.sh (hunk 3 above)
Comment (tagomaru) [reply launch_parity_local.sh]: Sorry, but i forgot to delete this file.

File: .gitlab-ci.yml (hunk 1 above)
Comment (tagomaru) [reply .gitlab-ci.yml]: Yes, sorry.

File: README.md (hunk 2 above)
Comment (tagomaru) [reply README.md:6]: ok

**Review by mrtizmoatwork — APPROVED** [review]
ok

**Review by Rizato — CHANGES_REQUESTED** [review]
Need to make the changes from this priv-testnet commit here. 

* adds a config.toml to start sealing blocks
* modified genesis.json to start in POA mode with a block per second
* sets up genesis.json to use tabs (it was mostly tabs, but some spaces)

https://github.com/polyswarm/priv-testnet/pull/7/commits/96fe052526f3893d48f2948f9cfeeffc4012ef86

The max size change I made in that commit is a mistake, though.

**Review by Rizato — APPROVED** [review]
lgtm

**Pull request merged**